File: docs/components_page/components/form/grid.jl
using DashBootstrapComponents

form = dbc_row(
    [
        dbc_col(
            [
                dbc_label("Email", html_for = "example-email-grid"),
                dbc_input(
                    type = "email",
                    id = "example-email-grid",
                    placeholder = "Enter email",
                ),
            ],
            width = 6,
        ),
        dbc_col(
            [
                dbc_label("Password", html_for = "example-password-grid"),
                dbc_input(
                    type = "password",
                    id = "example-password-grid",
                    placeholder = "Enter password",
                ),
            ],
            width = 6,
        ),
    ],
    className = "g-3",
)

File: docs/components_page/components/form/grid.py
import dash_bootstrap_components as dbc

form = dbc.Row(
    [
        dbc.Col(
            [
                dbc.Label("Email", html_for="example-email-grid"),
                dbc.Input(
                    type="email",
                    id="example-email-grid",
                    placeholder="Enter email",
                ),
            ],
            width=6,
        ),
        dbc.Col(
            [
                dbc.Label("Password", html_for="example-password-grid"),
                dbc.Input(
                    type="password",
                    id="example-password-grid",
                    placeholder="Enter password",
                ),
            ],
            width=6,
        ),
    ],
    className="g-3",
)
